spec/usecases/reactions/update_materials_spec.rb

```diff
@@ -120,5 +135,19 @@
         expect(Sample.find_by(name: 'product').target_amount_value).to eq(99.08304)
       end
     end
+
+    context 'when vessel volume is valid' do
+      it 'returns the calculated mole gas product' do
+        result = class_instance.send(:update_mole_gas_product, product_sample, 80)
+        expect(result).to eq(3.5543368129550247e-06)
```

Comment:
Is this high of a precision relevant? If not, maybe reduce precision to reduce potential for this test to break.

Reply:
This is related to a legacy code of the NumeralInputWithUnitsCompo component which needs to be refactored in a separate PR.

Reply:
Is the refactoring necessary to reduce the numerical precision in the test?

Reply:
yes, I guess so